src/index.html

```diff
@@ -13,6 +13,59 @@
     />
```

Comment:
Keep your attributes correctly formatted. Since the link element has more than 2 attributes, put each attribute on a new line with proper indentation.